src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student2/BookController2.java

```diff
@@ -0,0 +1,15 @@
+package com.xpanxion.java.springboot.da1.demo.controller.student2;
+
+import com.xpanxion.java.springboot.da1.demo.model.student2.Book;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class BookController2 {
+    Book book = new Book();
+
+    @GetMapping("student2/api/v1/book")
+    public String getMessage() {
+        return book.getBook();
```

Comment:
Instead of: return book.getBook();

Let's change this to something like:

Book book = new Book("Harry Potter", "111-11-111-1111");
return book;

Don't forget to add "getters" to the book class.